stubs/crontab/_crontab.pyi

```diff
@@ -0,0 +1,22 @@
+import datetime
```

Comment:
It would work without this. This is a typing stub, which is a way to add typing information for Python modules. A lot of third party modules are collected in the [typeshed](https://github.com/python/typeshed) project, but this one is not among them so I added it here so mypy knows the methods' signatures.